### DISCUSSION TITLE
Connecting to pgAdmin

### DISCUSSION THREAD

**Discussion by dschinkel** [oguimbal/pg-mem#16] [General]
What would the connection settings be if I wanted to look at the in-memory DB through a tool like pgAdmin?  Or is this not possible?  I know there is a visual tool out there for viewing sqlite, would be nice to do the same for this one.

**Comment by oguimbal** [comment]
Nope, I see no straightforward way to do that, since there is no networking involved.

And I think this is a bit ouf of scope, since there is no actual persistence involved in pg-mem. I dont see how you could inspect a db that is fundamentally transient, and which is just erased from memory when your test finishes.

What is your use case ?
